docs/quick_start/AA_hexane_ethanol_example.rst

```diff
@@ -0,0 +1,97 @@
+Example of an All-Atom (AA) Hexane and Ethanol System
+========================
+
+**Note**: `foyer <https://foyer.mosdef.org/en/stable/>`_ is used in conjunction with **mBuild** in the following example to demonstrate how all the **MoSDeF** software integrates to generate all the required to execute simulations.
+
+Import the required mbuild package.
+
+.. code:: ipython3
+
+    import mbuild as mb
+
+
+Construct an all-atom (AA) hexane and ethanol using the OPLS-AA force field (FF),
+which is comes as a standard `foyer <https://foyer.mosdef.org/en/stable/>`_ force field (FF).
+The hexane and ethanol residues will be named HEX and ETO, respectively.  
+Lastly, the hexane and ethanol molecule's configuration will be energy minimized. 
+
+**Note**: The energy minimize step does also require the `foyer <https://foyer.mosdef.org/en/stable/>`_ package. 
+
+.. code:: ipython3
+
+    hexane = mb.load('CCCCCC', smiles=True, name='HEX')
+    hexane.name = 'HEX'
+    hexane.energy_minimize(forcefield='oplsaa' , steps=10**4)
+
+
+    ethanol = mb.load('CCO', smiles=True)
+    ethanol.name = 'ETO'
+    ethanol.energy_minimize(forcefield='oplsaa' , steps=10**4)
+
+
+Build the liquid box designed for 300 K using and isobaric-isothermal (NPT) simulation.
+The liquid box is built to a density of 680 kg/m^3, with a 50/50 mol ratio of hexane and ethanol, 
+and will be an orthogonal box measuring 5.0 nm in all the x, y, and z-dimensions.
+
+.. code:: ipython3
+
+    box_liq = mb.fill_box(compound= [hexane, ethanol],
+                          density=680,
+                          compound_ratio=[0.5, 0.5],
+                          box=[5.0, 5.0, 5.0])
+
+
+
+Writing all the GOMC files
```

Comment:
I think this doesn't belong in an _mBuild_ quickstart... Although very useful for getting started with GOMC, not everyone using mBuild will be using GOMC. If I were a new user and saw engine-specific instructions (for an engine I don't use) in the quickstart guide, it would deter me.

To me mBuild is for building engine-agnostic chemical systems and has the capability to save these systems to various formats. I think this should be moved to Tutorials so as not to give the idea that mBuild and GOMC are more connected than any other engine.

Even better might be to create and link to a mosdef-GOMC package, similar to what @rsdefever has done with mosdef-cassandra: [docs](https://mosdef-cassandra.readthedocs.io/en/latest/),  [repo](https://github.com/MaginnGroup/mosdef_cassandra)

Reply:
My goal is to show a undergrad/1st year grad to get up and running a very basic setup with a start-to-finish quick examples, with all the simulation engines included (GOMC is in there now but I want all to be).  I believe we need this to gain new user traction.  However, the way to get thru the whole example with all the engine specific parts is up for debate (no real straight forward solution...).  Maybe this Quickstart examples needs its only location outside of mBuild?


The top box build is meant for any engine.  We can link the engine specific example to finish, but we need others to add their engines... 


Currently, I'm more of a fan of keeping all the packages in one location.  It may be fine for our dev team to have stuff in different locations (we can easily ask each other), but I think it is hard for others outside the group to know the stuff exists, where to find it ....

Reply:
I see what Jenny is saying, if the goal of this quick start tutorial to concisely highlight the functionality and capabilities of mBuild, then it probably shouldn't go any further than creating a compound and showing how it can be saved to a file (.mol2, .xyz. pdb, etc..).  

If the goal is to show how all of the tools in the mosdef packages can be used in sucession to initialize and run simulations, then that's a beyond the scope of mBuild, and I think would fit more so under a "MosDeF Quick Start Guide"

> Maybe this Quickstart examples needs its only location outside of mBuild?
Like you mentioned, we could have a separate guide that shows how to use the built-in engine specific writers to go from a system created in mBuild to various simulation engines. This guide could be linked to from any tutorials made for mBuild and foyer. 

> My goal is to show a undergrad/1st year grad to get up and running a very basic setup with a start-to-finish quick examples, with all the simulation engines included (GOMC is in there now but I want all to be)

I think these kinds of examples would be great, but I don't think this should be the first thing people see when looking at an mBuild quick start guide.

Reply:
After some though, I think a good place for this would be https://github.com/mosdef-hub/mosdef-workflows --this is where a lot of engine-specific stuff goes.

Reply:
deleted the intro, and all items in the Quickstart after building the box